lib/profiles/MEM-SUBM.js

```diff
@@ -1,7 +1,7 @@
 
 exports.name = "MEM-SUBM";
 exports.config = {
-    status:             "SUBM"
+    status:             "MEM-SUBM"
```

Comment:
OK!
I see both [member](https://www.w3.org/Submission/) and [team](https://www.w3.org/TeamSubmission/) submissions use `SUBM`. Shall I correct the `status` of `lib/profiles/TEAM-SUBM.js`, too (I set it to `TEAM-SUBM`)? (Is it OK that both profiles share the same *short status*?)